WorkoutTest/ProgramViewController.swift

```diff
@@ -60,6 +60,19 @@ extension ProgramViewController: UITableViewDataSource {
     return programCell
   }
   
+  func tableView(_ tableView: UITableView, trailingSwipeActionsConfigurationForRowAt indexPath: IndexPath) -> UISwipeActionsConfiguration? {
+    let deleteProgram = UIContextualAction(style: .destructive, title: "Delete") { _, _, completion in
+      
```

Comment:
all is good, but i'd rename this property to something like 'let deleteProgramAction'.